.github/ISSUE_TEMPLATE/gitmoji-proposal.yml

```
@@ -47,12 +47,14 @@ body:
         - "No ❌"
     validations:
       required: true
-  - type: textarea
+  - type: checkboxes
     id: emoji-what-how
     attributes:
       label:  Does this emoji fall into the "how" category?
-      description: Explain the creation of this emoji, what this is and when it should be used
-      placeholder: We are trying to always describe/categorize the "what" has been done in one particular commit (:white_check_mark:, :lock:, :zap:...), not the "how" it was done (the exceptions being :hankey:, :beers:)
+      description: We are trying to always describe/categorize the "what" has been done in one particular commit (:white_check_mark:, :lock:, :zap:...), not the "how" it was done (the exceptions being :hankey:, :beers:)
```

I think we can remove this:

```suggestion
      description: We are trying to always describe/categorize "what" has been done in one particular commit (:white_check_mark:, :lock:, :zap:...), not "how" it was done (the exceptions being :hankey:, :beers:)
```